---
float32[2] center
float32 inner_radius
float32 outer_radius
